examples/test_resnet_ptq_cpu.py

```diff
@@ -30,7 +30,7 @@ def check_output(footprints):
     for footprint in footprints.values():
         assert footprint.nodes is not None
         for v in footprint.nodes.values():
-            assert all([value > 0 for value in v.metrics.value.values()])
+            assert all([value.value > 0 for value in v.metrics.value.values()])
```

Comment:
so many value, metrics.value.values().value. Is it a little bit wired?

Reply:
Updated to increase readability.